src/include/executor/plan_executor.h

```diff
@@ -28,43 +28,37 @@ namespace executor {
 // Plan Executor
 //===----------------------------------------------------------------------===//
 
-typedef struct ExecuteResult {
-  peloton::ResultType m_result;
-  int *m_result_slots;
+struct ExecuteResult {
```

Comment:
This name is a little unintuitive since it just captures data about an execution, not actually executing anything. How about changing it to `ExecutionResult`?